### ISSUE TITLE
Letters are cropped

### ISSUE THREAD

**Issue by krzfls** [otfried/ipe#106]
_Issue:_ Some letters are cropped (seems to be similar to #93).
_In more detail:_ In pdf, some letters (e.g., "$p$") written in text labels are not displayed completely. Some parts of the letters are cut away; see example below. 
_Observed in:_ IPE 7.2.6. (on Ubuntu, Windows 7 and OS X) and also on previous IPE versions.

_Example:_
![example_p](https://cloud.githubusercontent.com/assets/23058304/19694589/58eee7ec-9ab6-11e6-9714-912ad2a3b8a9.png)
In this example, the left bottom part of $p$ is missing. Also note that the boudning box is higher than the actual letter.

_Related oservations:_
In IPE, the selection box is smaller than the actual letter for most letters. Maybe there is a connection to this issue.
The cropping also happens, when the bounding box of the picture is bigger than the letter.

**Comment by otfried** [comment]
You can fix this like in #105: add a style sheet like this:

```
<ipestyle name="pad">
<textpad left="1" right="1" top="1" bottom="1"/>
</ipestyle>
```

The default left pad was `0`, because I hadn't noticed that some fonts contain letters that stick out on the left as well.  I guess we need to make the default `1` for all four directions.

**Comment by krzfls** [comment]
Thanks for the fix!
